ui/pages/permissions-connect/choose-account/index.scss

```diff
@@ -1,4 +1,5 @@
 .permissions-connect-choose-account {
+  position: fixed;
```

Comment:
Actually, I'm not certain we want this. This will end up placing the site being connected to right up against the top of the screen. Trying to figure out an alternative.